frontend/src/components/camp/Picasso.vue

```diff
@@ -477,7 +450,6 @@ export default {
   min-width: 20px !important;
   top: 0 !important;
   right: 0 !important;
-  opacity: 0;
```

Comment:
Open to discussion, enabled it to route to the activity page.

Reply:
While I was trying out your PR, I noticed that it would probably be better if this button opened the popup and when you click on ScheduleEntry in Picasso, it immediately switches to the Activity view.

Reply:
That would make the double click action obsolete, correct?

Reply:
Yes